python/foglamp/services/core/server.py

```diff
@@ -177,7 +178,8 @@ def _start_core(cls, loop=None):
             # see http://<core_mgt_host>:<core_mgt_port>/foglamp/service for registered services
 
             _logger.info('Announce management API service')
-            cls.management_announcer = ServiceAnnouncer('FogLAMP-Core', '_foglamp_core._tcp', cls.core_management_port,
+            unique_id = str(uuid.uuid4())
+            cls.management_announcer = ServiceAnnouncer('FogLAMP-Core-{}'.format(unique_id), '_foglampcore._tcp', cls.core_management_port,
```

Comment:
It's important that we have the same name each time we start, therefore the use of a changing uuid is not the best approach. the intention is to allow the name to be configured, with the default being FogLAMP. This configuration item would replace the FogLAMP portion of the current name.